### ISSUE TITLE
Add back person thumbnail links, but hide them from screen readers

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/tiamat-theme#1716]
I worded this ticket totally wrong: https://github.com/CuBoulder/tiamat-theme/issues/1702

Instead, we should have kept the links but added the following: role="presentation" aria-hidden="true"

This would have addressed the accessibility issue, while still leaving the thumbnails for people to use in navigating the site.  For reference, this is what we did with the thumbnails on articles.
